_episodes/07-find.md

```diff
@@ -249,6 +249,34 @@ Miscellaneous:
 > {: .solution}
 {: .challenge}
 
+> ## More `grep` practice
+>
+> What would be the correct command for the of the following text :
```

Comment:
It's not clear what the question is asking. Aside from fixing the existing wording, perhaps stating in words what pattern the learner should search for, show the desired result (as you already have done) and ask the learner to choose the right command would be a better question format?